crates/services/src/service.rs

```diff
@@ -218,7 +218,7 @@ where
     }
 }
 
-#[tracing::instrument(skip_all)]
+#[tracing::instrument(skip_all, fields(service = S::NAME))]
```

Comment:
This is cool! Didn't realize this was an available option to annotate all the logs from a struct instance.